EasyPost.Tests/CalculationTests/Rates.cs

```diff
@@ -0,0 +1,132 @@
+using System.Collections.Generic;
+using EasyPost.Exceptions.General;
+using EasyPost.Models.API;
+using EasyPost.Tests._Utilities.Annotations;
+using Xunit;
+
+namespace EasyPost.Tests.CalculationTests
+{
+    public class Rates
+    {
+        #region Tests
+
+        [Fact]
+        [Testing.Function]
+        public void TestGetLowestRateObject()
+        {
+            List<Rate> rates = new()
+            {
+                new Rate { Price = "100.00" },
+                new Rate { Price = "1.00" }
+            };
+
+            Rate lowestRate = Calculation.Rates.GetLowestObjectRate(rates);
+            Assert.Equal("1.00", lowestRate.Price);
+        }
+
+        [Fact]
+        [Testing.Function]
+        public void TestGetLowestShipmentSmartrate()
+        {
+            List<Smartrate> rates = new()
+            {
+                new Smartrate
+                {
+                    Rate = 100.00,
+                    DeliveryDays = 1,
+                    TimeInTransit = new TimeInTransit { Percentile50 = 1 }
+                },
+                new Smartrate
+                {
+                    Rate = 1.00,
+                    DeliveryDays = 4,
+                    TimeInTransit = new TimeInTransit { Percentile50 = 4 }
+                }
+            };
+
+            Smartrate lowestRate = Calculation.Rates.GetLowestShipmentSmartrate(rates, 1, SmartrateAccuracy.Percentile50);
```

Comment:
Thought: may want to specify here that the accuracy is taken into account which is why the higher of the two prices is returned (future travelers may wonder by scanning why the lower rate wasn't used)

Reply:
Resolved